kafka-client/src/main/java/dev/responsive/kafka/internal/db/rs3/client/grpc/GrpcRs3Util.java

```diff
@@ -201,4 +204,32 @@ public static Optional<Long> walOffsetFromProto(final Rs3.WALOffset walOffset) {
       return Optional.empty();
     }
   }
+
+  public static UUID uuidFromProto(final Rs3.UUID uuidProto) {
```

Comment:
We have this defined in `dev.reponsive.kafka.utils.Utils` as well. I think this is a better location. Maybe we can remove the other definition?